source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/organizations/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import List from './list';
+import AddOrganizationForm from './new';
+import EditOrganizationForm from './edit';
+
+import { Switch, Route } from 'react-router-dom';
+
+export const listPathName = '/admin/settings/organizations';
+export const addOrganizationPathName = '/admin/settings/organizations/new';
+export const editOrganizationPathName = '/admin/settings/organizations/:orgId/edit';
+
+class OrganizationRoute extends React.Component {
+
+  render() {
+    return (
+      <div className='sub-page-content' data-test-admin-organizations>
+        <Switch>
+          <Route exact path={listPathName} render={(routeProps) => (
```

Comment:
why render instead of component + withRouter in the HoC chain?

Reply:
When I started doing this page this component has more html in it. What improvement will we get to change it to hoc

Reply:
your basically saying all of these components need the routeProps.

it's better for the component to decide whether or not it needs access to the route (deciding if it needs a router consumer)